introspection_request_handler.go

```diff
@@ -121,7 +121,9 @@ func (f *Fosite) NewIntrospectionRequest(ctx context.Context, r *http.Request, s
 			return &IntrospectionResponse{Active: false}, errors.Wrap(ErrRequestUnauthorized, "HTTP Authorization header missing, malformed or credentials used are invalid")
 		}
 	}
-
+	// if r.PostForm.Get("migrated") == "true" {
```

Comment:
planning on removing this commented code?

Reply:
Yes